packages/opentelemetry-instrumentation-milvus/opentelemetry/instrumentation/milvus/__init__.py

```diff
@@ -0,0 +1,93 @@
+"""OpenTelemetry Milvus DB instrumentation"""
+
+import logging
+import pymilvus
+
+from typing import Collection
+
+from opentelemetry.instrumentation.milvus.config import Config
+from opentelemetry.trace import get_tracer
+from wrapt import wrap_function_wrapper
+
+from opentelemetry.instrumentation.instrumentor import BaseInstrumentor
+from opentelemetry.instrumentation.utils import unwrap
+
+from opentelemetry.instrumentation.milvus.wrapper import _wrap
+from opentelemetry.instrumentation.milvus.version import __version__
+
+logger = logging.getLogger(__name__)
+
+_instruments = ("pymilvus >= 2.4.1",)
+
+WRAPPED_METHODS = [
+    {
+        "package": pymilvus,
+        "object": "MilvusClient",
+        "method": "insert",
+        "span_name": "milvus.insert"
+    },
+    {
+        "package": pymilvus,
```

Comment:
I think this needs to be a string otherwise this may fail

Reply:
Hi! From what I saw in the chromadb instrumentation the package is added directly as this, I don't know if for this case it's different but it seems to work as this?

Reply:
You're right! I was only looking at Pinecone